cmd/api/book/book_test.go

```diff
@@ -54,13 +43,22 @@ func TestCreateBook(t *testing.T) {
 			return b, nil
 		})
 
+		wg := sync.WaitGroup{}
+		wg.Add(1)
+		mockNtfy.EXPECT().BookCreated(gomock.Any(), reqBook.Name, *reqBook.Inventory).DoAndReturn(func(ctx context.Context, title string, inventory int) error {
+			defer wg.Done()
+			return errors.New("error to test ntfy return")
```

Comment:
This error will not be actually tested, because after return, it is just logged with `log.Println` . I even tried to capture de log at stdout, but then I thought that I was going to far to this simple test. Don't you think?

Reply:
It was to be an error that I would try to catch at the log at stdout. But now that we decided not going so far, this error realy don't make sense anymore. I'll change the error to nil and assume it is a success.